Frontend/src/pages/admin/AdminCreateNotificationPage.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+  <BasePage title="Send Notification">
+    <q-card>
+      <div class="q-pa-md">
+        <q-table
+          title="Select Students"
+          :data="tableStudents"
+          :columns="columns"
+          row-key="email"
+          selection="multiple"
+          :selected.sync="toSendTo"
+        />
+        <q-input
+          outlined
+          v-model="title"
+          label="Title"
+          class="notifPageComponents"
+        />
+        <q-input
+          v-model="body"
+          label="Body"
+          outlined
+          type="textarea"
+          class="notifPageComponents"
+        />
+        <q-btn
+          v-if="!sending"
+          color="primary"
```

Comment:
Add `:disabled="sending"` here to disable the button while it's sending

Reply:
it doesnt show up when its sending tho